tokens.json

```diff
@@ -1846,13 +1846,29 @@
         "chain": "eos"
     },
     {
-        "name": "Yup",
-        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUPX.png",
-        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUPX.png",
+        "name": "YUPX",
+        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
         "symbol": "YUPX",
         "account": "yupyupxtoken",
         "chain": "eos"
     },
+    {
+        "name": "YUP",
+        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "symbol": "YUP",
+        "account": "token.yup",
+        "chain": "eos"
+    },
+    {
+        "name": "YUPETH LP",
+        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "symbol": "YUPETH LP",
+        "account": "lptoken.yup",
```

Comment:
token contract is not deployed to this account

Reply:
token contract will be ready by end of day

Reply:
We can't accept tokens that don't yet exist, let us know when it does.

Reply:
got it, no problem. I submitted a PR without the lp token for now, could that be approved for now?

Reply:
Also your token symbol is invalid, it can’t have 8 characters, the maximum limit is 7 characters

Reply:
@DenisCarriere got it, thank you. I just removed the yupeth lp token entry entirely. Is this looking good now?